### PR TITLE
 Changing the Timeout from Minute to Seconds, and minor bug fixes. 

### PR DESCRIPTION
**Pull request by harneetvirk**
Changing the Timeout from Minute to Seconds, and minor bug fixes.

### CODE REVIEW

File: src/main/java/com/microsoft/kafkaavailability/App.java [hunk 1]
```diff
@@ -158,7 +158,9 @@ private static void waitForChanges(CuratorManager curatorManager) throws Excepti
 
     private static void RunOnce(CuratorFramework curatorFramework) throws IOException, MetaDataManagerException {
 
-        HeartBeat beat = new HeartBeat(appProperties.environmentName);
+        //default to 1 minute, if not configured
+        long heartBeatIntervalInSeconds = (appProperties.heartBeatIntervalInSeconds > 0 ? appProperties.heartBeatIntervalInSeconds : 60000);
```
Comment (soumyajit-sahu) [inline src/main/java/com/microsoft/kafkaavailability/App.java]: 60000 secs?

File: src/main/java/com/microsoft/kafkaavailability/App.java [hunk 2]
```diff
@@ -209,8 +211,8 @@ protected boolean onAdvance(int phase, int registeredParties) {
         //default to 5 minutes, if not configured
         long consumerThreadSleepTime = (appProperties.consumerThreadSleepTime > 0 ? appProperties.consumerThreadSleepTime : 300000);
 
-        //default to 15 minutes, if not configured
-        long mainThreadsTimeoutInMinutes = (appProperties.mainThreadsTimeoutInMinutes > 0 ? appProperties.mainThreadsTimeoutInMinutes : 900000);
+        //default to 10 minutes, if not configured
+        long mainThreadsTimeoutInSeconds = (appProperties.mainThreadsTimeoutInSeconds > 0 ? appProperties.mainThreadsTimeoutInSeconds : 600000);
```
Comment (soumyajit-sahu) [inline src/main/java/com/microsoft/kafkaavailability/App.java]: s/600000/600/

File: src/main/java/com/microsoft/kafkaavailability/threads/ConsumerThread.java [hunk 3]
```diff
@@ -56,47 +56,43 @@ public ConsumerThread(Phaser phaser, CuratorFramework curatorFramework, List<Str
...
+        }
+        elapsedTime = CommonUtils.stopWatch(lStartTime);
+        m_logger.info("Consumer Elapsed: " + elapsedTime + " milliseconds.");
 
+        try {
             m_phaser.arriveAndDeregister();
-            CommonUtils.dumpPhaserState("After arrival of ConsumerThread", m_phaser);
-
-        } while (!m_phaser.isTerminated());
+        } catch (IllegalStateException success) {
```
Comment (soumyajit-sahu) [inline src/main/java/com/microsoft/kafkaavailability/threads/ConsumerThread.java]: rename variable name success to something better

File: src/main/java/com/microsoft/kafkaavailability/App.java (hunk 1 above)
Comment (harneetvirk) [inline src/main/java/com/microsoft/kafkaavailability/App.java]: This is changed to 60

File: src/main/resources/appProperties.json [hunk 4]
```diff
@@ -23,5 +23,6 @@
   "reportInterval": 0,
   "consumerPartitionTimeoutInSeconds": 30,
   "consumerTopicTimeoutInSeconds": 60,
-  "mainThreadsTimeoutInMinutes": 900000
+  "mainThreadsTimeoutInSeconds": 600000,
```
Comment (soumyajit-sahu) [inline src/main/resources/appProperties.json]: Similar correction here too. Reduce the seconds

File: src/main/java/com/microsoft/kafkaavailability/App.java (hunk 2 above)
Comment (harneetvirk) [inline src/main/java/com/microsoft/kafkaavailability/App.java]: This is changed to 600

File: src/main/java/com/microsoft/kafkaavailability/threads/ConsumerThread.java [hunk 5]
```diff
@@ -56,47 +56,43 @@ public ConsumerThread(Phaser phaser, CuratorFramework curatorFramework, List<Str
...
         int sleepDuration = 1000;
         long elapsedTime = 0L;
-        do {
-            long lStartTime = System.nanoTime();
-            MetricRegistry metrics;
-            m_logger.info(Thread.currentThread().getName() +
-                    " - Consumer party has arrived and is working in "
-                    + "Phase-" + m_phaser.getPhase());
-
+        long lStartTime = System.nanoTime();
```
Comment (soumyajit-sahu) [inline src/main/java/com/microsoft/kafkaavailability/threads/ConsumerThread.java]: Let us use System.currentTimeMillis() here

**Pull request merged**